tequilapi/endpoints/sessions.go

```diff
@@ -111,7 +112,7 @@ func sessionToDto(se session.Session) SessionDTO {
 		ProviderID:      string(se.ProviderID.Address),
 		ServiceType:     se.ServiceType,
 		ProviderCountry: se.ProviderCountry,
-		DateStarted:     se.Started.Format("2018-10-29 16:22:05"),
+		DateStarted:     se.Started.Format(time.RFC3339),
```

Comment:
You'll be able to parse the RFC3339 in any language easily. It does provide timezone information. One thing that unix timestamp is lacking is the human readability. RFC3339 is both human comprehendible and machine readable.

Reply:
I would say its a good time to introduce UTC unix timestamp, since we are changing format anyways. Will reduce complexity for users later on.

Reply:
@zolia golden rule with time - never ever do direct arithmetics with times. https://en.wikipedia.org/wiki/Leap_second

Reply:
I think the RFC3339 is still the way to go. You said it youself - peer will have to parse the timestamp to make sense of it. Why have that step as extra? If you're parsing anyway, you'll be able to parse RFC3339 with the same ease as a timestamp.

IETF seems to recommend RFC3339 as well:
https://tools.ietf.org/html/rfc7493#section-4.3

Reply:
Ok, theres cons and pros for that. RFC3339 is representative format as common JSON is too. I see a bunch of pitfalls in such usage, but since event IETF recommends that.. well :)